.github/workflows/build-sw-container.yml

```diff
@@ -0,0 +1,49 @@
+name: Build and Push Docker image
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - /^v([0-9.]+)$/
+
+jobs:
+  make-date-tag:
+    runs-on: ubuntu-latest
+    if: startsWith(github.repository, 'opensciencegrid/')
+    outputs:
+      dtag: ${{ steps.mkdatetag.outputs.dtag }}
+    steps:
+    - name: make date tag
+      id: mkdatetag
+      run: echo "::set-output name=dtag::$(date +%Y%m%d-%H%M)"
+
+  build:
+    runs-on: ubuntu-latest
+    if: startsWith(github.repository, 'opensciencegrid/')
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Generate tag list
+      id: generate-tag-list
+      env:
+        REPO: ${{ matrix.repo }}
+      run: |
+        docker_repo=${GITHUB_REPOSITORY/opensciencegrid\/docker-/opensciencegrid/}
+        tag_list=latest,${{ GITHUB_REF##v }}
```

Comment:
```suggestion
        tag_list=$docker_repo:latest,$docker_repo:${GITHUB_REF#v }
```

1. I forgot to mention that each tag has the form `CONTAINER REPO:CONTAINER TAG`
2. In bash, you can reference vars with `$VAR` or `${VAR}`
3. We can slightly shorten the parameter expansion since we just need to remove the single character. From `man bash`:

```
       ${parameter#word}
       ${parameter##word}
              Remove  matching  prefix  pattern.   The word is expanded to produce a pattern just as in pathname expansion, and matched against the expanded value of parameter using the
              rules described under Pattern Matching below.  If the pattern matches the beginning of the value of parameter, then the result of the expansion is the  expanded  value  of
              parameter with the shortest matching pattern (the ``#'' case) or the longest matching pattern (the ``##'' case) deleted.  If parameter is @ or *, the pattern removal oper‐
              ation is applied to each positional parameter in turn, and the expansion is the resultant list.  If parameter is an array variable subscripted with @ or *, the pattern re‐
              moval operation is applied to each member of the array in turn, and the expansion is the resultant list.
```